client/app/components/create/createService.js

```diff
@@ -3,7 +3,7 @@ angular.module('create.factory', [])
     var createProject = function(data) {
       return $http({
         method: 'POST',
-        url: '/create',
+        url: '/api/users',

```

Comment:
Might be something I'm missing, but why have we changed the URL here to /api/users?